# points: a list of each trajectory point that is needed to hit

hw2/TrajectoryCommand[] points
